hepcrawl/spiders/common/oaipmh_spider.py

```diff
@@ -47,51 +48,63 @@ class OAIPMHSpider(LastRunStoreSpider):
     __metaclass__ = abc.ABCMeta
     name = 'OAI-PMH'
 
+    @strict_kwargs
     def __init__(
         self,
         url,
         format='oai_dc',
         sets=None,
-        alias=None,
+        identifier=None,
         from_date=None,
         until_date=None,
-        *args, **kwargs
+        **kwargs
     ):
-        super(OAIPMHSpider, self).__init__(*args, **kwargs)
+        super(OAIPMHSpider, self).__init__(**self._all_kwargs)
         self.url = url
-        self.format = format
         if isinstance(sets, string_types):
-            sets = sets.split(',')
-        self.sets = sets
-        self.from_date = from_date
-        self.until_date = until_date
```

Comment:
I thin this should be kept. I don't like 'hiding' them in the `_init_kwargs` variable, it's too implicit.